requirements.txt

```diff
@@ -4,16 +4,18 @@
 # with package version changes made in requirements-app.txt
 
 aws-wsgi==0.2.7
-boto==2.49.0
+boto3==1.17.58
```

Comment:
It didn't make sense to use boto 2 and boto 3 side by side, so we got rid of the deprecated version!